components-ui/button/index.tsx

```diff
@@ -1,8 +1,7 @@
 'use client';
```

Comment:
No need to use it as a client component anymore ?

Reply:
I think we do, as there is an event callback forwarding (onClick). I'll check on that.

Reply:
Confirmed here: https://react.dev/reference/react/use-client#limitations